app/helpers/vm_helper/textual_summary.rb

```diff
@@ -333,7 +333,7 @@ def textual_flavor
 
   def textual_service
     h = {:label => _("Service"), :image => "service"}
-    service = @record.service
+    service = @record.service || (@record.respond_to?(:orchestration_stack) && @record.orchestration_stack.try(:service))
```

Comment:
aah, cool, they moved that behavior to try!, seems like my knowledge of try was still rails 3 :-D

thank you, I will change that